source/includes/steps-install-mongodb-on-amazon-tarball.yaml

```diff
@@ -0,0 +1,60 @@
+title: Download the MongoDB ``.tgz`` tarball.
```

Comment:
General thought - these steps *seem* pretty generic, on pass 2 we might single-source these steps in particular.